providers/indexing.csv

```diff
@@ -1,10 +1,32 @@
+ormi-free,Ormi Labs,null,Subgraph Indexing,Free,FALSE,$0.00,$0.00,FALSE,"[""GraphQL"",""REST API""]","[""TPS: 20 req/10s"",""100k entities included""]","[""Encryption and monitoring""]","[""Real-time analytics""]","[""Global""]","[""Email support""]",NULL,NULL,NULL,,,,,
+ormi-pay-as-you-go,Ormi Labs,null,Subgraph Indexing,Pay-as-you-go,FALSE,$4 per 100k entities stored,$0.00,FALSE,"[""GraphQL"",""REST API""]","[""TPS: 50 req/10s""]","[""Encryption and monitoring""]","[""Real-time analytics""]","[""Global""]","[""Premium support via Email, Slack, and Telegram""]",NULL,NULL,NULL,,,,,
+ormi-enterprise,Ormi Labs,null,Subgraph Indexing,Enterprise,FALSE,$1 per 100k entities stored,$0.00,FALSE,"[""GraphQL"",""REST API""]","[""Custom""]","[""Encryption and monitoring""]","[""Real-time analytics""]","[""Global""]","[""Uptime and latency SLA"",""Optional dedicated environment""]",NULL,NULL,NULL,,,,,
+sqd-free,SQD Cloud,null,Decentralized Data Lakes,Free,FALSE,$0.00,$0.00,FALSE,"[""GraphQL"",""SQL""]","[""1 squid""]","[""Decentralized storage"",""IP Protection""]","[""Data lake logs"",""Usage analytics""]","[""Global""]","[""ETL query stack"",""Decentralized indexing"",""Discord support""]",NULL,NULL,NULL,,,,,
+sqd-pay-as-you-go,SQD Cloud,null,Decentralized Data Lakes,Pay-as-you-go,FALSE,$86+ (depending on squid needs may go up to $1.5k and higher),$0.00,FALSE,"[""GraphQL"",""SQL""]",,"[""Decentralized storage"",""IP Protection""]","[""Data lake logs"",""Usage analytics""]","[""Global""]","[""ETL query stack"",""Decentralized indexing"",""Hard SLA with compensations""]",NULL,NULL,NULL,,,,,
+subquery-free,SubQuery,null,Decentralized Data Indexing,Free,FALSE,$0.00,$0.00,FALSE,"[""GraphQL"",""SQL""]","[""Rate limits apply""]","[""Decentralized storage"",""IP Protection""]","[""Data lake logs"",""Usage analytics""]","[""Global""]","[""ETL query stack"",""Decentralized indexing"",""Discord support""]",NULL,NULL,NULL,,,,,
+subquery-pay-as-you-go,SubQuery,null,Decentralized Data Indexing,Pay-as-you-go,FALSE,,,FALSE,"[""GraphQL"",""SQL""]","[""Rate limits may apply""]","[""Decentralized storage"",""IP Protection""]","[""Data lake logs"",""Usage analytics""]","[""Global""]","[""ETL query stack"",""Decentralized indexing"",""Hard SLA with compensations""]",NULL,NULL,NULL,,,,,
+subquery-enterprise,SubQuery,null,Decentralized Data Indexing,Enterprise,FALSE,,,FALSE,"[""GraphQL"",""SQL""]",,"[""Decentralized storage"",""IP Protection""]","[""Data lake logs"",""Usage analytics""]","[""Global""]","[""ETL query stack"",""Decentralized indexing"",""Hard SLA with compensations""]",NULL,NULL,NULL,,,,,
+thegraph-free,The Graph,null,"Subgraphs, Substreams",Free,FALSE,$0.00,$0.00,FALSE,"[""GraphQL""]","[""Up to 100k queries/month""]","[""Decentralized infra""]","[""Subgraph logs"",""Usage analytics""]","[""Global""]","[""Hosted indexing"",""real-time data updates"" ]",NULL,NULL,NULL,,,,,
```

Comment:
Good point that the graph is having Substreams now. Maybe I would suggest to keep "The Graph" as technology and move "Substream" to the additionalFeatures?